build.gradle

```diff
@@ -55,58 +49,31 @@ task checkstyle(type: Checkstyle) {
     source = sourceSets.main.allJava + sourceSets.test.allJava
 }
 
-task gem(type: JRubyExec, dependsOn: ["gemspec", "classpath"]) {
-    jrubyArgs "-S"
-    script "gem"
-    scriptArgs "build", "${project.name}.gemspec"
-    doLast { ant.move(file: "${project.name}-${project.version}.gem", todir: "pkg") }
-}
 
-task gemPush(type: JRubyExec, dependsOn: ["gem"]) {
-    jrubyArgs "-S"
-    script "gem"
-    scriptArgs "push", "pkg/${project.name}-${project.version}.gem"
-}
-
-task "package"(dependsOn: ["gemspec", "classpath"]) {
-    doLast {
-        println "> Build succeeded."
-        println "> You can run embulk with '-L ${file(".").absolutePath}' argument."
-    }
+embulkPlugin {
+    mainClass = "org.embulk.input.jira.JiraInputPlugin"
+    category = "input"
+    type = "jira"
 }
 
 jacocoTestReport {
     group = "Reporting"
     reports {
         xml.enabled false
         csv.enabled false
-        html.destination "${buildDir}/reports/coverage"
+        html.destination file("${buildDir}/reports/coverage")
     }
 }
 
-task gemspec {
-    ext.gemspecFile = file("${project.name}.gemspec")
-    inputs.file "build.gradle"
-    outputs.file gemspecFile
-    doLast { gemspecFile.write($/
-Gem::Specification.new do |spec|
-  spec.name          = "${project.name}"
-  spec.version       = "${project.version}"
-  spec.authors       = ["uu59", "yoshihara"]
-  spec.summary       = %[Jira input plugin for Embulk]
-  spec.description   = %[Loads records from Jira.]
-  spec.email         = ["k@uu59.org", "h.yoshihara@everyleaf.com"]
-  spec.licenses      = ["Apache-2.0"]
-  spec.homepage      = "https://github.com/treasure-data/embulk-input-jira"
 
-  spec.files         = `git ls-files`.split("\n") + Dir["classpath/*.jar"]
-  spec.test_files    = spec.files.grep(%r"^(test|spec)/")
-  spec.require_paths = ["lib"]
+gem {
+    authors = ['uu59', 'yoshihara']
+    email = ['k@uu59.org', 'h.yoshihara@everyleaf.com']
+    summary = project.description
+    homepage = 'https://github.com/treasure-data/embulk-input-jira'
+    licenses = ['Apache-2.0']
+}
 
-  spec.add_development_dependency 'bundler', ['~> 1.0']
-  spec.add_development_dependency 'rake', ['~> 12.0']
-end
-/$)
-    }
+gemPush {
+    host = "https://treasuredata.jfrog.io/treasuredata/api/gems/rubygems-local"
```

Comment:
This host is not valid. Please use `https://rubygems.org/`

Reply:
Good catch, thanks!